packages/infra/README.md

```
@@ -0,0 +1,127 @@
+# Esposter Infrastructure
+
+[![Apache-2.0 licensed][badge-license]][url-license]
+
+Pulumi infrastructure code and migration tooling for Esposter Azure resources.
+
+### Table of Contents
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)